app/views/govuk_publishing_components/components/_layout_header.html.erb

```diff
@@ -1,15 +1,17 @@
 <%
-  product_name ||= "Publishing"
-  public_environment = environment.eql?("public")
+  product_name ||= false
+  environment ||= false
```

Comment:
Minor I feel like nil is better representative than false for these. False seems to apply they could be true which would be wrong, whereas nil encapsulates a lack of value.

Reply:
I'd wondered about this! Thanks for the clarity. I felt that `nil` might have been better but couldn't verbalise it properly. Change.